src/cmd/projectImport.go

```diff
@@ -14,16 +15,16 @@ import (
 	"github.com/zeropsio/zerops-go/types/uuid"
 )
 
-const projectImportArgName = "importYamlPath"
+const defaultYamlFilePattern = "*import.yml"
 
 func projectImportCmd() *cmdBuilder.Cmd {
 	return cmdBuilder.NewCmd().
-		Use("project-import").
+		Use("import-project").
```

Comment:
don't change command's use for backwards compatibility

Reply:
Alright